src/components/Clock.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export class Clock extends React.Component {
+  state = {
```

Comment:
You can describe type of State as in the App.tsx

Reply:
Thanks, done